tools/odps_table_tools/transform_kv_table.py

```diff
@@ -0,0 +1,318 @@
+import argparse
+import time
+
+import odps
+from odps import options, ODPS
+
+options.log_view_host = "http://logview.odps.aliyun-inc.com:8080"
```

Comment:
This information can be exposed to public?

Reply:
We can find the information in the document of aliyun. https://help.aliyun.com/document_detail/42745.html?spm=5176.10695662.1996646101.searchclickresult.4d975c33YqRrs8
So, I think it can be exposed to the public.